assignments/2. variables/index.md

```diff
@@ -1,35 +1,36 @@
-1. In code below "Mark" is a string.  What is name?
+//Solution in the same file
+1. In code below "Mark" is a string.  What is name?//string
 ```js
 var name = "Mark";
 ```
 
 2. Find the error if any
 ```js
-  var product cost = 3.45;
+  var product cost = 3.45;//var product-cost=3.45
 ```
 
 3. Write `Right or Wrong` next to the code below.
 
 ```js
-  "Hello World"
-  'Hello World"
-  "Hello World'
-  'Hello World'
+  "Hello World"//right
+  'Hello World"//wrong
+  "Hello World'//wrong
+  'Hello World'//right
 ```
 
 ## Write `VALID` and `INVALID` infront of the variable name defined below
 ```js
-var man;
-var 1girl;
-var woman3;
-var -girl;
-var blackDog;
-var 42;
-var $42;
-var userName;
-var x, y, z;
-var x = 5, y = 6, z = 7;
-var x = 5 + 10 + 2;
+var man;//invalid
+var 1girl;//invalid
+var woman3;//invalid
+var -girl;//invalid
+var blackDog;//invalid
+var 42;//invalid
+var $42;//invalid
+var userName;//invalid
+var x, y, z;//invalid
```

Comment:
Read this chapter again and refactor the code. http://javascript.info/variables

Reply:
done